packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/TestHelpers/MockHelpers.cs

```diff
@@ -16,11 +16,12 @@ internal static class MockHelpers
 
         public static void LoadMockPlugin(
             Func<InputType, CSharpType>? createCSharpTypeCore = null,
-            Func<OutputLibrary>? createOutputLibrary = null)
+            Func<OutputLibrary>? createOutputLibrary = null,
+            string? configuration = null)
         {
             var configFilePath = Path.Combine(AppContext.BaseDirectory, TestHelpersFolder);
             // initialize the singleton instance of the plugin
-            var mockPlugin = new Mock<CodeModelPlugin>(new GeneratorContext(Configuration.Load(configFilePath))) { CallBase = true };
+            var mockPlugin = new Mock<CodeModelPlugin>(new GeneratorContext(Configuration.Load(configuration is null ? configFilePath : String.Empty, configuration))) { CallBase = true };
```

Comment:
Should outputPath param be nullable, and possibly optional?

Reply:
this api is awkward because the two overloads would both take a string.  One is the string path and one is the string json.  Perhaps we can just have two named methods instead.

Reply:
actually i just got rid of the string empty special handling it always requires a valid path.